### PR TITLE
chore: Backport artifactory endpoint changes to v1.7 

### LINKED ISSUE #14581: v1.7 and v1.8 are using the lf artifactory for installing magma
### 1.7, 1.8 using magma-packages

* [x] test 1.7, 1.8 install with sed script in install.sh
  * between checkout and install
  * 1.7 force merge needed
  * 1.8 OK from govind/yogesh needed
  * 1.7 needs proto fix (?)
* [x] merge this
  * [x] 1.7
  * [x] 1.8
* [x] communicating in #dev (?) that 1.7 and 1.8 install now use the new artifactory
* [x] asking lf for the virt repo

### PR DESCRIPTION
**Pull request by ajahl**
## Summary

This PR is a backport of #14508. Since the magmacore artifactory will be switched off at the end of November, the PR migrates relevant artifactory endpoints  from `magmacore.org` to `linuxfoundation.jfrog.io`.

## Test Plan

- [x] Tested on a blank Ubuntu installation with `agw_install_ubuntu.sh`

## Additional Information

The PR can only be force-merged since the CI is already broken for v1.7 and thus cannot run through.

- [ ] This change is backwards-breaking

### CODE REVIEW

File: .github/workflows/amis-workflow.yml [hunk 1]
```diff
@@ -57,7 +57,7 @@ jobs:
               echo PACKAGE_VERSION="${GIT_BRANCH_VERSION:1}" >> $GITHUB_ENV
               echo VERSION="${GIT_BRANCH_VERSION:1}" >> $GITHUB_ENV
             fi
-            echo PACKAGE_REPO_HOST="artifactory.magmacore.org\\\/artifactory\\\/debian" >> $GITHUB_ENV
+            echo PACKAGE_REPO_HOST="linuxfoundation.jfrog.io\\\/artifactory\\\/magma-packages-test" >> $GITHUB_ENV
```
Comment (sebathomas) [inline .github/workflows/amis-workflow.yml]:
```suggestion
            echo PACKAGE_REPO_HOST="linuxfoundation.jfrog.io\\\/artifactory\\\/magma-packages" >> $GITHUB_ENV
```

File: orc8r/tools/ansible/roles/pkgrepo/tasks/debian.yml [hunk 2]
```diff
@@ -36,11 +31,11 @@
...
 - name: Add JFrog key
   apt_key:
-    url: https://facebookconnectivity.jfrog.io/artifactory/api/gpg/key/public
+    url: https://linuxfoundation.jfrog.io/artifactory/api/security/keypair/magmaci/public
     state: present
 
 - name: Add JFrog repo
   apt_repository:
-    repo: 'deb https://facebookconnectivity.jfrog.io/artifactory/list/{{ repo }}/ {{ distribution }} main'
+    repo: 'deb https://linuxfoundation.jfrog.io/artifactory/magma-packages focal-1.7.0 main'
```
Comment (sebathomas) [inline orc8r/tools/ansible/roles/pkgrepo/tasks/debian.yml]: Can we change the variables in the corresponding file instead?

File: xwf/gateway/deploy/roles/pkgrepo/tasks/debian.yml [hunk 3]
```diff
@@ -11,19 +11,14 @@
...
   apt: pkg=apt-transport-https state=present update_cache=yes
 
 - name: Add JFrog key
-  shell: wget -qO - https://facebookconnectivity.jfrog.io/artifactory/api/gpg/key/public | apt-key add -
+  shell: wget -qO - https://linuxfoundation.jfrog.io/artifactory/api/security/keypair/magmaci/public | apt-key add -
 
 - name: Add JFrog repo
   apt_repository:
-    repo: 'deb https://facebookconnectivity.jfrog.io/artifactory/list/{{ repo }}/ {{ distribution }} main'
+    repo: 'deb https://linuxfoundation.jfrog.io/artifactory/magma-packages focal-1.7.0 main'
```
Comment (sebathomas) [inline xwf/gateway/deploy/roles/pkgrepo/tasks/debian.yml]: Same here.

File: .github/workflows/amis-workflow.yml (hunk 1 above)
Comment (ajahl) [reply .github/workflows/amis-workflow.yml]: github action is removed

**Review by wolfseb — APPROVED** [review]
lgtm

**Review by MoritzThomasHuebner — APPROVED** [review]
LGTM

File: .github/workflows/build_all.yml [hunk 4]
```diff
@@ -575,7 +575,7 @@ jobs:
         id: publish_artifacts
         if: github.event_name == 'push'
         env:
-          DOCKER_REGISTRY: "cwf-test.artifactory.magmacore.org"
+          DOCKER_REGISTRY: "linuxfoundation.jfrog.io/magma-docker-cwf-test"
```
Comment (voisey) [inline .github/workflows/build_all.yml]: I think this should be `magma-docker-cwag-test`

Comment (voisey) [reply .github/workflows/build_all.yml]: See https://linuxfoundation.jfrog.io/ui/repos/tree/General/magma-docker-cwag-test

File: .github/workflows/helm-deploy-on-demand.yml [hunk 5]
```diff
@@ -7,7 +7,7 @@ on:  # yamllint disable-line rule:truthy
 jobs:
   build_publish_helm_charts_on_demand:
     env:
-      HELM_CHART_ARTIFACTORY_URL: "https://artifactory.magmacore.org:443/artifactory/"
+      HELM_CHART_ARTIFACTORY_URL: " https://linuxfoundation.jfrog.io/artifactory/"
```
Comment (voisey) [inline .github/workflows/helm-deploy-on-demand.yml]: Space added here by mistake

File: xwf/gateway/deploy/roles/pkgrepo/tasks/debian.yml [hunk 6]
```diff
@@ -11,19 +11,14 @@
...
   apt: pkg=apt-transport-https state=present update_cache=yes
 
 - name: Add JFrog key
-  shell: wget -qO - https://facebookconnectivity.jfrog.io/artifactory/api/gpg/key/public | apt-key add -
+  shell: wget -qO - https://linuxfoundation.jfrog.io/artifactory/api/security/keypair/magmaci/public | apt-key add -
 
 - name: Add JFrog repo
   apt_repository:
-    repo: 'deb https://facebookconnectivity.jfrog.io/artifactory/list/{{ repo }}/ {{ distribution }} main'
+    repo: 'deb https://linuxfoundation.jfrog.io/artifactory/{{ repo }} {{ distribution }} main'
```
Comment (voisey) [inline xwf/gateway/deploy/roles/pkgrepo/tasks/debian.yml:22]: Was the slash intentionally removed here?

Comment (ajahl) [reply xwf/gateway/deploy/roles/pkgrepo/tasks/debian.yml:22]: The slash is not required here.

**Review by panyogesh — APPROVED** [review]
LGTM

File: .github/workflows/deploy-build-from-pr.yml [hunk 7]
```diff
@@ -130,7 +130,7 @@ jobs:
       - name: Tag and push cwag docker images to Regitry
         if: always() && steps.extract_images.outcome=='success'
         env:
-          DOCKER_REGISTRY: "cwf-ci.artifactory.magmacore.org"
+          DOCKER_REGISTRY: "linuxfoundation.jfrog.io/artifactory/magma-docker-cwf-test"
```
Comment (voisey) [inline .github/workflows/deploy-build-from-pr.yml]: I think this should be cwag-test too

File: xwf/gateway/deploy/roles/pkgrepo/tasks/debian.yml (hunk 6 above)
Comment (voisey) [reply xwf/gateway/deploy/roles/pkgrepo/tasks/debian.yml:22]: Sorry, I thought that the distribution was part of the link but now I see that it's not

**Review by jheidbrink — APPROVED** [review]

**Review by voisey — APPROVED** [review]

**Review by sebathomas — APPROVED** [review]

**Pull request merged**